.github/workflows/continuous-integration.yml

```diff
@@ -1,66 +1,108 @@
 name: Continuous integration
 on: pull_request
 jobs:
+  metadata:
+    name: Metadata
+    runs-on: ubuntu-latest
+    outputs:
+      python_headline_version: ${{ steps.list-python-versions.outputs.python_headline_version }}
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
```

Comment:
I take it 3.8.9 was not an option in github actions?

Reply:
It's an option, but we don't need to run on an older version of Python for this stage. So it's one less thing to try and fix up.